docs/virtual_machines/interfaces_and_networks.md

```diff
@@ -479,7 +479,7 @@ fields.
 More information about SLIRP mode can be found in [QEMU
 Wiki](https://wiki.qemu.org/Documentation/Networking#User_Networking_.28SLIRP.29).
 
-### masquerade
+### masquerade - IPv4 single-stack support
```

Comment:
Is the `port` section only relevant for IPv4 single stack? This is the impression now, as it resides under this subject.

Reply:
It is relevant to all the cluster types, IPv4 only. dual stack and IPv6 only. The example yaml referenced here contains port. Isn't it enough? Do you think it should be explicitly mentioned?

Reply:
But you added `- IPv4 single-stack support`, making it specific.

Reply:
you're right. I will remove the `IPv4 single-stack support`